utils/src/main/scala/za/co/absa/enceladus/utils/types/TypedStructField.scala

```diff
@@ -123,6 +126,8 @@ sealed abstract class TypedStructField(structField: StructField)(implicit defaul
 }
 
 object TypedStructField {
+  private val logger = LoggerFactory.getLogger(this.getClass)
```

Comment:
Logger not needed, not used

Reply:
True, I have been logging something and refrained from doing so afterward, will fix, thanks.